src/api.ts

```diff
@@ -72,8 +72,14 @@ router.post('/', async (req, res) => {
     });
   }
 
+  const msg = req.body.data.message;
+  if (!req.body.data.types.Space && !msg.settings && !msg.space) {
```

Comment:
maybe better to check schema like on sequencer? 🤔

Reply:
We could certainly do that, it's outside this PR scope. I'll open another issue for that